### PR TITLE
chore: move custom field of quotation from bloomstack_core to erpnext

### PR DESCRIPTION
**Pull request by meshramsaroj**
**Task Link:** https://bloomstack.com/desk#Form/Task/TASK-2021-00249

**Issue:** bloomstack_core/bloomstack_core/bloomstack_core/custom/quotation.json

**Description:** Moved required field of quotation doctype from bloomstack core to erpnext and also deleted quotation.json from bloomstack_core

**Deleted file PR link:** https://github.com/Bloomstack/bloomstack_core/pull/647

**Screenshot:**

- Before

1. In ERPNext
image

2. In Bloomstack core
image

- After
1. In ERPNext
image

2. In Bloomstack core
image

### CODE REVIEW

**Review by sahil28297 — CHANGES_REQUESTED** [review]
customer_signature and signature fields are missing

File: erpnext/selling/doctype/quotation/quotation.json [hunk 1]
```diff
@@ -91,6 +93,7 @@
   "terms",
   "signature_section_section",
   "signature",
```
Comment (meshramsaroj) [inline erpnext/selling/doctype/quotation/quotation.json:95]: @sahil28297
signature is already there, I have added the customer signature field.
Please check

File: erpnext/selling/doctype/quotation/quotation.json [hunk 2]
```diff
@@ -951,13 +953,30 @@
...
+  {
+   "fieldname": "license",
+   "fieldtype": "Link",
+   "label": "License",
+   "options": "Compliance Info"
+  },
+  {
+   "fieldname": "customer_signature",
+   "fieldtype": "Signature",
+   "label": "Customer Signature"
```
Comment (sahil28297) [inline erpnext/selling/doctype/quotation/quotation.json]: "read_only": 1, "print_hide": 1,

**Review by sahil28297 — CHANGES_REQUESTED** [review]

File: erpnext/selling/doctype/quotation/quotation.json (hunk 2 above)
Comment (meshramsaroj) [reply erpnext/selling/doctype/quotation/quotation.json]: added required changes

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**